tests/discovery.rs

```diff
@@ -813,7 +813,7 @@ mod proxy_to_proxy {
         env.put(app::config::ENV_TLS_PRIVATE_KEY, key);
         env.put(app::config::ENV_TLS_TRUST_ANCHORS, trust_anchors);
         env.put(
-            app::config::ENV_TLS_POD_IDENTITY,
+            app::config::ENV_TLS_local_identITY,
```

Comment:
I think the casing is messed up here

Reply:
Yeah, this looks like a case-insensitive find-and-replace accident or something.